book/views.py

```diff
@@ -44,27 +45,35 @@ def post(self, request, *args, **kwargs):
 
 class OrderView(View):
     def get(self, request, *args, **kwargs):
-        email = request.GET.get('user')
-        books = Books.objects.all()
-        return render(request, 'order_book.html', {'books':books, 'user':email})
+        uid = request.GET.get('u_id')
+        user = CustomUser.objects.filter(id=uid).first()
+        bname = request.GET.get('b_name')
```

Comment:
For book details, Use the book_id instead of passing details in the request params